src/CLI/Generator.cs

```diff
@@ -6,155 +6,158 @@
             {
                 messages.Add("Input library name not specified and check symbols not enabled. Either set the input library name or the check symbols flag");
                 return false;
             }
 
-            if (_options.InputLibraryName == String.Empty && _options.CheckSymbols == true && _options.Libraries.Count == 0)
+            if (string.IsNullOrEmpty(options.InputLibraryName) && options.CheckSymbols && options.Libraries.Count == 0)
             {
                 messages.Add("Input library name not specified and check symbols is enabled but no libraries were given. Either set the input library name or add at least one library");
                 return false;
             }
+            
+            StringBuilder tripleBuilder = new StringBuilder();
+            if (options.Architecture == TargetArchitecture.x64)
+                tripleBuilder.Append("x86_64-");
+            else if(options.Architecture == TargetArchitecture.x86)
+                tripleBuilder.Append("i686-");
 
-            if (_options.Architecture == TargetArchitecture.x64)
-                _triple = "x86_64-";
-            else if(_options.Architecture == TargetArchitecture.x86)
-                _triple = "i686-";
-
-            if (_options.Platform == TargetPlatform.Windows)
+            if (options.Platform == TargetPlatform.Windows)
             {
-                _triple += "pc-win32-msvc";
-                _abi = CppAbi.Microsoft;
+                tripleBuilder.Append("pc-win32-msvc");
+                abi = CppAbi.Microsoft;
             }
-            else if (_options.Platform == TargetPlatform.MacOS)
+            else if (options.Platform == TargetPlatform.MacOS)
             {
-                _triple += "apple-darwin12.4.0";
-                _abi = CppAbi.Itanium;
+                tripleBuilder.Append("apple-darwin12.4.0");
+                abi = CppAbi.Itanium;
             }
-            else if (_options.Platform == TargetPlatform.Linux)
+            else if (options.Platform == TargetPlatform.Linux)
             {
-                _triple += "linux-gnu";
-                _abi = CppAbi.Itanium;
+                tripleBuilder.Append("linux-gnu");
+                abi = CppAbi.Itanium;
 
-                if(_options.Cpp11ABI)
-                    _triple += "-cxx11abi";
+                if(options.Cpp11ABI)
+                    tripleBuilder.Append("-cxx11abi");
             }
 
+            triple = tripleBuilder.ToString();
+
             return true;
         }
 
         public void Setup(Driver driver)
         {
             var parserOptions = driver.ParserOptions;
-            parserOptions.TargetTriple = _triple;
-            parserOptions.Abi = _abi;
+            parserOptions.TargetTriple = triple;
+            parserOptions.Abi = abi;
 
             var options = driver.Options;
-            options.LibraryName = _options.OutputFileName;
+            options.LibraryName = this.options.OutputFileName;
```

Comment:
Some more "this" to remove. To make sure, please search for all usages of the field.